interview/backend/.dockerignore

```diff
@@ -0,0 +1,5 @@
+.idea
+.vscode
+*/**.pyc
```

Comment:
I think we need to unifiy this with .gitignore **/*.pyc

Reply:
Agreed.

Reply:
Agreed, done.

Reply:
ok